tests/numerics/generalized_interpolation_02.output

```diff
@@ -4,61 +4,61 @@ DEAL::Function value at (0.0,0.0): 1.00000
 DEAL::Check projection property: 0.00000
 DEAL::dim 2 FE_Q<2>(2)
 DEAL::Function value at (0.0,0.0): 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 2.22045e-16
 DEAL::dim 3 FE_Q<3>(3)
 DEAL::Function value at (0.0,0.0): 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 8.88178e-16
 DEAL::dim 2 FE_Q<2>(1)
 DEAL::Function value at (0.0,0.0): 1.00000 
 DEAL::Check projection property: 0.00000
 DEAL::dim 2 FE_Q<2>(2)
 DEAL::Function value at (0.0,0.0): 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 2.22045e-16
 DEAL::dim 3 FE_Q<3>(3)
 DEAL::Function value at (0.0,0.0): 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 8.88178e-16
 DEAL::dim 2 FE_RaviartThomas<2>(0)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 5.55112e-17
 DEAL::dim 2 FE_RaviartThomas<2>(1)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 2.35922e-16
 DEAL::dim 2 FE_RaviartThomas<2>(2)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 0.169271
 DEAL::dim 3 FE_RaviartThomas<3>(0)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 1.11022e-16
 DEAL::dim 3 FE_RaviartThomas<3>(1)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 2.91434e-16
 DEAL::dim 2 FE_RaviartThomas<2>(0)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 1.11022e-16
 DEAL::dim 2 FE_RaviartThomas<2>(1)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 4.44089e-16
 DEAL::dim 2 FE_RaviartThomas<2>(2)
 DEAL::Function value at (0.0,0.0): 1.00000 1.00000 
-DEAL::Check projection property: 0.00000
+DEAL::Check projection property: 0.169442
```

Comment:
Do you have an explanation for this (large) change? The same for a few entries below for RT and Nedelec cases.

Reply:
@masterleinad seems convinced that these tests should not fail, but I'm not so sure about this. I'll try to understand a bit better, and if they are not supposed to work, I'll disable them.

The explanation I found is that a point that was previously found on a given cell is now on the neighbor cell, and we are testing discontinuous shape functions on generalized support points. Since this can happen, using a FEFieldFunction on DG elements should not be considered robust w.r.t. to a function beeing single valued.

On the other hand, @masterleinad suggested that the function which is tested here *is* actually continuous, so this should not fail. I'll try to investigate a bit more.